app/src/main/java/com/kickstarter/viewmodels/EnvironmentalCommitmentsViewHolderViewModel.kt

```diff
@@ -23,8 +25,7 @@ interface EnvironmentalCommitmentsViewHolderViewModel {
         fun category(): Observable<Int>
     }
 
-    class EnvironmentalCommitmentsViewHolderViewModel() :
-        ViewModel(), Inputs, Outputs {
```

Comment:
Just curious why the viewmodel extension was removed here, if its no longer a viewmodel should this just be a util class?